src/UI/App.jsx

```diff
@@ -304,12 +307,12 @@ export default class App extends React.Component {
 		}
 
 		// Check working hours when they changed
-		if(nextState.workingHours !== this.state.workingHours) {
+		if(JSON.stringify(nextState.workingHours) !== JSON.stringify(this.state.workingHours)) {
```

Comment:
`workingHours` is als het goed is een array en is dit dus niet nodig.
Was de bestaande check problematisch en heb je dat bewezen in een test? Anders is de json stringify hier niet nodig.

Reply:
Ik heb een bug gevonden bij workinghours. Op dit moment gaat deze altijd goed ook met stringify. Dit komt als je bijvoorbeeld in de console de working hours update:

window.parleySettings.weekdays =  [
['Monday', 0.00, 23.59],
['Tuesday', 0.00, 23.59],
['Wednesday', 0.00, 23.59],
['Thursday', 0.00, 23.59],
['Friday', 0.00, 23.59],
['Saturday', 0.00, 23.59],
['Sunday', 0.00, 23.59],
]

en dit vervolgens weer doet, zou je verwachten dat hij de nieuwe waardes tot zich opneemt. Inplaats daarvan vult hij de huidige array die hij had aan waardoor het dubbel komt te staan. Op deze manier vind altijd een verandering plaats. Zal ik dit opnemen in een ander bug ticket?

Reply:
> Zal ik dit opnemen in een ander bug ticket?

Nee het is prima om hierin te fixen, maar ik kon me niet voorstellen dat `array !== array` niet werkte.
Daarnaast mis ik ook een test die bewijst dat je fix werkt.

Reply:
Nog even snel getest en je hebt gelijk 
![image](https://github.com/parley-messaging/web-library/assets/26112463/0056bc84-f0c1-4131-a60e-15e9b5255193)

Dus je moet nog even een Cypress test maken die dit issue test en zorgt dat ie bij een regressie weer af gaat

Reply:
Net besproken en er zit nog een fout in het mergen van de oude workinghours state met de nieuwe, dus we negeren deze setting voor nu en fixen dat in een ander ticket (bouke zet het ticket hierbij)